### DISCUSSION TITLE
Trouble with 23.9.3 Drivers?

### DISCUSSION THREAD

**Discussion by pw405** [lshqqytiger/stable-diffusion-webui-directml#287] [General]
All - I upgraded to Radeon 23.9.3 drivers today, as the patch notes specifically mention enhancements for Stable Diffusion!

> New Feature Highlights
> ...
> MeanVarianceNormalization (MVN) and Stable Diffusion variants optimizations

However, I've had nothing but trouble since updating.  Earlier in the day I was generating batches of 4 SDXL images at 1216 X 832, 30 steps (Juggernaut V5 Model, RealVisXL 2, etc.).  No problems really.  But now I'm getting out of VRAM crashes after ONE image on a 7900XTX.  I'm going to reinstall drivers, etc, but I was curious if others had similar experiences?

**Comment by pw405** [comment]
Update!  I went nuts last night with OS updates, App updates, Motherboard software updates, yadda yadda, ran some Windows maintenance batch files, etc.  I re-installed Radeon Drivers afterwards and tested a little bit earlier today.  Seems stable, and there is a respectable speed increase - where I was seeing about 1.8 seconds per iteration, I now see about 1.2 seconds per iteration, so that's a decent little speed up. 

I'll update once I know more.

**Reply by ratunku** [reply]
Tested the new driver yesterday. Runs way better than older ones. Higher resolutions are now possible on my RX 7900 XTX (tried up to 1024x1024) and speeds above 512x512 are noticeably faster, depending on the resolution.

**Reply by pw405** [reply]
Nice! I've noticed the same as well. Maybe my OCD use of the AMD Bug Report tool every time I crash my system generating images is starting to pay off!  Hahaha, I must have submitted at least 20 bug reports after an SD crash by now.
